pubspec.yaml

```diff
@@ -72,6 +72,12 @@ dependencies:
   # receive_sharing_intent
   receive_sharing_intent: ^1.4.1
 
+  # intl
+  intl: ^0.16.1
```

Comment:
We use it, and other in somewhere. Can we define the version in one place? Like we did in `version.gradle` for android project.

Reply:
We need this for `DateFormat` in `DateTimeExtension`

Reply:
Yes, I see. I mean we centralize version definition in one place

Reply:
yes, I'm researching about it.

Reply:
So far I haven't seen a way to do it. I created a ticket for this because its quite out of scoped right now.
[https://ci.linagora.com/linagora/lgs/linshare/products/linshare-mobile-apps/issues/575](https://ci.linagora.com/linagora/lgs/linshare/products/linshare-mobile-apps/issues/575)